frontend/public/locales/en/translation.json

```diff
@@ -148,12 +148,12 @@
     "important": "Important",
     "crucial": "Crucial",
     "criteria": "Criteria",
-    "today": "Today",
-    "thisWeek": "This week",
-    "thisMonth": "This month",
-    "thisYear": "This year",
-    "allTime": "All time",
-    "uploadDate": "Upload date",
+    "today": "A day",
+    "thisWeek": "A week",
+    "thisMonth": "A month",
+    "thisYear": "A year",
+    "allTime": "Always",
+    "uploadDate": "Uploaded since",
```

Comment:
Probably not very important, but videos "uploaded since a week" doesn't sound like correct english to me. Contrary to something like "uploaded over the past week".

Reply:
"Uploaded for a month/week/year" would be the translation proposed by Google translate: https://translate.google.ch/?sl=fr&tl=en&text=Mises%20en%20ligne%20depuis%20un%20mois&op=translate